src/intro.c

```diff
@@ -34,7 +34,6 @@ void intro_update()
 {
 	rct_drawpixelinfo *screenDPI = &gScreenDPI;
 	int screenWidth = gScreenWidth;
```

Comment:
I think we can replace `screenWidth` with `gScreenWidth`. It's probably a leftover from when gScreenWidth was still an RCT2_GLOBAL.